Editor/AGS.Editor/Validation/String.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace AGS.Editor.Validation
+{
+    public static class String
```

Comment:
This can get awfully confusing seeing as it shares a name with an already existing class in the .NET API. If I write `String.IsNotEmpty(null)` how can I be super sure which type I'm working on without manually checking? If someone less experienced is working on AGS how many hours will they spend pulling their hairs out before they notice the difference? Luckily the methods in here are very generic so can get around this conundrum by converting the methods into extension methods. So referencing them could be done directly on the variable `var result = "some string".IsASCIICharactersOnly()`